system-int-test/test/cmr/system_int_test/search/facets/collection_facets_v2_search_test.clj

```diff
@@ -103,6 +112,15 @@
   (index/wait-until-indexed)
   (testing "No fields applied for facets"
     (is (= fr/expected-v2-facets-apply-links (search-and-return-v2-facets))))
+  (testing "Facets size applied for facets"
+    (is (= fr/expected-v2-facets-apply-links-with-facets-size 
+           (search-and-return-v2-facets {:facets-size {:platform 1}}))))
+  (testing "Empty facets size applied for facets"
```

Comment:
This test is not testing an Empty facets size.

Reply:
Sorry. I changed the data without changing a title. Will add another test. Done.